lib/capturable.js

```diff
@@ -11,16 +11,23 @@ module.exports = function capturable () {
     function _expr (value, args) {
         var captured = events;
         events = [];
+        var source = {
+            content: args.content,
+            filepath: args.filepath,
+            line: args.line
+        };
+        if (args.generator) {
+            source.generator = true;
+        }
+        if (args.async) {
+            source.async = true;
+        }
```

Comment:
@twada 
Is source the correct place to put these?
Or should they go on powerAssertContext?
I don't understand what the purpose of the two different objects are.